src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const costOfRent = 40;
+  const discountBasics = 20;
+  const discountExtra = 50;
+  let totalCost = costOfRent * days;
+
+  if (days % 3 === 0) {
+    return (totalCost -= discountBasics);
```

Comment:
Ohh, you don't need assignment operation here, don't use assignment operations in the `return`
```suggestion
    return totalCost - discountBasics;
```